database/associations.js

```diff
@@ -1,6 +1,7 @@
 module.exports = function(models) {
     const User = models.user_account;
     const Progress = models.progress;
+    const Infokit = models.info_kit;
     Progress.belongsTo(User, {foreignKey: 'user_id', targetKey: 'id', onDelete: 'cascade'});
+    Infokit.belongsTo(User, {foreignKey: 'user_id', targetKey: 'id', onDelete: 'cascade'});
```

Comment:
Infokit database has activity names to be shown in infokit, while progress shows the latest activity completed.  There are some actitivites which do not need an infokit element, and there can be some extra information points in infokit, so we had separated them.